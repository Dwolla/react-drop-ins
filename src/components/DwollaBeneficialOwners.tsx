import type { FC } from 'react';
import { BeneficialOwnersAttributes } from '../@types/component-attributes';

const DwollaBeneficialOwners: FC<BeneficialOwnersAttributes<any>> = (props) => {
  return <dwolla-customer-create {...props} />;
};

export default DwollaBeneficialOwners;
