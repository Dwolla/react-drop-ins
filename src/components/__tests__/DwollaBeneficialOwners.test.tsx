import TestRenderer from 'react-test-renderer';
import DwollaBeneficialOwners from '../DwollaBeneficialOwners';

test('DwollaBeneficialOwners', () => {
  const tree = TestRenderer.create(<DwollaBeneficialOwners customerId="123" />).toJSON();

  expect(tree).toMatchSnapshot();
});
