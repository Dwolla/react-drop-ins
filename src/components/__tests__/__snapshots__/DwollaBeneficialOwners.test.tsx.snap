// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DwollaBeneficialOwners 1`] = `
<dwolla-customer-create
  customerId="123"
/>
`;
